docs/machine-learning/tutorials/restaurant-violation-classification-model-builder.md

```diff
@@ -0,0 +1,168 @@
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/08/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to classify violation severity using Model Builder.
+---
+
+# Tutorial: Classify severity of restaurant violations with Model Builder
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+>
+> - Prepare and understand the data
+> - Choose a scenario
+> - Load the data
+> - Train the model
+> - Evaluate the model
+> - Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Pre-requisites
+
+For a list of pre-requisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Create a console application
+
+1. Create a **.NET Core Console Application** called "RestaurantViolationClassification".
+
+## Prepare and understand the data
+
+The data set used to train and evaluate the machine learning model is originally from the [San Francisco Department of Public Health Restaurant Safety Scores](https://www.sfdph.org/dph/EH/Food/score/default.asp).
+
+1. Download the [dataset](https://data.sfgov.org/api/views/pyih-qa8i/rows.csv?accessType=DOWNLOAD) and save it anywhere on your computer.
+
+1. Open the **Restaurant_Scores_-_LIVES_Standard.csv** data set
+
+    The provided data set contains several prefixed columns.
+    
+    The columns with the *business* prefix contain information about the business such as the address and phone number. These set of columns are not relevant for training a model. To make it more concrete, the phone number or zip code of an establishment does not determine what the severity of a violation is. 
+
+    The columns with the *inspection* prefix contain formation about individual inspections conducted on the business. A subset of these columns is relevant for building the model. The columns of interest are:
+
+    - **inspection_id** the unique ID associated with a specific violation
+    - **inspection_type** the type of inspection. This can either be a first-time inspection for a new establishment, a routine inspection, a complaint inspection, and many other types.
+    - **violation_description** a description of the violation encountered. This description corresponds to the **inspection_id**.
+
+The `label` is the column you want to predict. When performing a classification task, the goal is to assign a category (text or numerical). In this classification scenario, the severity of violation is assigned the value of low, medium or high risk. Therefore, the **risk_category** is the label. The identified `features` are the inputs you give the model to predict the `label`. In this case, the **inspection_id**, **inspection_type** and **violation_description** are used as features or inputs to predict the **risk_category**.
```

Comment:
Might be good to do a table or screenshot preview of the data. Do we use all of the columns for features? Can we show feature selection?

Reply:
#resolved